lib/message/decrypt.js

```diff
@@ -91,13 +91,19 @@ export async function decryptMessageLegacy(options) {
  * @return {Promise<{getBody: (function(): Promise<{body, mimetype}>), getAttachments: (function(): Promise<any>), getEncryptedSubject: (function(): Promise<any>), verify: (function(): Promise<any>), errors: (function(): Promise<any>), stop: stop}>}
  */
 export async function decryptMIMEMessage(options) {
-    const { data: rawData, verified: embeddedVerified, errors } = await decryptMessageLegacy(options);
-
-    const { body, mimetype, verified: pgpVerified, attachments, encryptedSubject } = await processMIME(
-        options,
-        rawData
+    const { data: rawData, verified: embeddedVerified, errors, signatures: pgpSignatures } = await decryptMessageLegacy(
+        options
     );
 
+    const {
+        body,
+        mimetype,
+        verified: pgpVerified,
+        attachments,
+        encryptedSubject,
+        signatures: mimeSignatures
+    } = await processMIME(options, rawData);
```

Comment:
Your new names (`pgpSignatures` and `mimeSignatures`) are correct, but in combination with the previous ones (`embeddedVerified` and `pgpVerified`) it's very confusing.
I would propose `verified` and `signatures` for the first set, and `pgpMimeVerified` and `pgpMimeSignatures` for the second one.